### PR TITLE
i18n(fr): Translation of `cms.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Added page translation of `cms.mdx`. This is my first attempt of translating docs to french. All suggestions are much appreciated.

### CODE REVIEW

File: src/pages/fr/guides/cms.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Guides CMS
+
+<CMSGuidesNav />
+
+Notez que plusieurs de ces pages sont des **ébauches**: ce sont des collections de ressources en attente de votre contribution!
+
+## Pourquoi utiliser un CMS?
+
+Une Système de Gestion de Contenu vous permettez écrire du contenu et gérer les actifs hors de votre projet Astro.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/cms.mdx:20]:
```suggestion
Un Système de Gestion de Contenu vous permet d'écrire du contenu et de gérer les actifs en dehors de votre projet Astro.
```

File: src/pages/fr/guides/cms.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Cela ouvre de nouvelles fonctionnalités pour travailler avec du contenu. La plupart des CMSes vous donnent un éditeur de contenu visuel, la possibilité de spécifier des types de contenu standard et un moyen de collaborer avec d'autres.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/cms.mdx:22]:
```suggestion
Cela ouvre de nouvelles fonctionnalités pour travailler avec du contenu. La plupart des CMS vous donnent un éditeur de contenu visuel, la possibilité de spécifier des types de contenu standard et un moyen de collaborer avec d'autres.
```
Nit: In French, I don't think that this type of acronym is pluralized

File: src/pages/fr/guides/cms.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Un CMS peut être utile pour un contenu qui suit un structure particulière, vous donnant souvent une expérience similaire à un tableau de bord et des outils d'édition WYSIWYG. Vous pouvez utiliser un CMS pour écrire des billets de blog en utilisant l'éditeur de texte riche d'un CMS au lieu de fichiers Markdown. Ou vous pouvez utiliser un CMS pour mantenir des listes de produits pour un magasin de commerce électronique, rendant champs requis pour éviter les inscriptions incomplètes.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/cms.mdx:24]:
```suggestion
Un CMS peut être utile pour un contenu qui suit un structure particulière, vous donnant souvent une expérience similaire à un tableau de bord et des outils d'édition WYSIWYG. Vous pouvez utiliser un CMS pour écrire des billets de blog en utilisant l'éditeur de texte riche d'un CMS au lieu de fichiers Markdown. Ou vous pourriez utiliser un CMS pour pour maintenir les listes de produits pour une boutique en ligne, en rendant certains champs obligatoires pour éviter des annonces incomplètes.
```

File: src/pages/fr/guides/cms.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Votre projet Astro peut alors récuperer votre contenu de votre CMS et l'afficher, où et vous le souhaitez sur votre site.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/cms.mdx:26]:
```suggestion
Votre projet Astro peut alors récuperer votre contenu de votre CMS et l'afficher, où et comment vous le souhaitez sur votre site.

File: src/pages/fr/guides/cms.mdx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## Quels CMS fonctionnent bien avec Astro?
+
+Parce qu'Astro s'occupe de la _présentation_ de votre contenu, vous devrez choisir un CMS _headless_ comme ceux de la liste ci-dessus. Cela signifie que le CMS vous aide à écrire votre contenu, mais ne génère pas un site qui l'affiche. Au lieu de cela, vous récupérez les données de contenu et les utilisez dans votre projet Astro.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/cms.mdx:30]:
```suggestion
Parce qu'Astro s'occupe de la _présentation_ de votre contenu, vous voudrez choisir un CMS _headless_ (découplé) comme ceux de la liste ci-dessus. Cela signifie que le CMS vous aide à écrire votre contenu, mais ne génère pas un site qui l'affiche. Au lieu de cela, vous récupérez les données de contenu et les utilisez dans votre projet Astro.
```

File: src/pages/fr/guides/cms.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+Certains CMSes headless, comme Storyblok, fournissent une [intégration](/fr/guides/integrations-guide) d'Astro qui aide à récupérer le contenu spécifiquement pour un site Astro. D'autres fournissent un SDK JavaScript, une bibliothèque que vous installez et utilisez pour récupérer votre contenu distant.
```
Comment (MoustaphaDev) [inline src/pages/fr/guides/cms.mdx:32]:
```suggestion
Certains CMS headless, comme Storyblok, fournissent une [intégration](/fr/guides/integrations-guide/) d'Astro qui aide à récupérer le contenu spécifiquement pour un site Astro. D'autres fournissent un SDK JavaScript, une bibliothèque que vous installez et utilisez pour récupérer votre contenu distant.
```
Should fix the link issue

**Review by MoustaphaDev — COMMENTED** [review]
This is looking good @dreyfus92! I just have a couple of suggestions

**Comment by dreyfus92** [comment]
@MoustaphaDev Merci pour le review 💚

**Review by thomasbnt — APPROVED** [review]

**Review by MoustaphaDev — APPROVED** [review]

**Pull request merged**